tests/Endpoints/Users/WebhookSubscriptions/SubscribedActivityTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\EoneoPay\PhpSdk\Endpoints\Users\WebhookSubscriptions;
+
+use EoneoPay\PhpSdk\Endpoints\Users\WebhookSubscriptions\SubscribedActivity;
+use Tests\EoneoPay\PhpSdk\TestCase;
+
+/**
+ * @covers \EoneoPay\PhpSdk\Endpoints\Users\WebhookSubscriptions\SubscribedActivity
+ */
+class SubscribedActivityTest extends TestCase
```

Comment:
Where is the test for something being in the array? Should test 0,1,n case with arrays

Reply:
`uris()` is always returning empty array as there are no actions directly on the `SubscribedActivity`. It is created via `WebhookSubscription`. So added the activity test on `WebhookTest`.